protoc-gen-connect-kotlin/src/main/kotlin/build/buf/protocgen/connect/Generator.kt

```diff
@@ -48,17 +52,19 @@ import com.squareup.kotlinpoet.asClassName
  * move off of type aliases, this can be changed without user API breakage.
  */
 private val HEADERS_CLASS_NAME = ClassName("build.buf.connect", "Headers")
+private val CANCELLABLE_CLASS_NAME = ClassName("build.buf.connect.http", "Cancelable")
```

Comment:
this has both `cancellable` and `cancelable` - consistency would be good 😄

Reply:
It's worth noting that [Android uses one L](https://developer.android.com/reference/android/app/Dialog#setCancelable(boolean)) but [Kotlin uses two L's](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines.flow/cancellable.html)